src/pkg/tinyspline-config.cmake.in

```diff
@@ -4,10 +4,8 @@ set_and_check(TINYSPLINE_LIBRARY_DIRS "@PACKAGE_TINYSPLINE_INSTALL_LIBRARY_DIR@"
 set(TINYSPLINE_VERSION "@TINYSPLINE_VERSION@")
 set(TINYSPLINE_DEFINITIONS "@TINYSPLINE_DEFINITIONS@")
 
-find_library(TINYSPLINE_LIBRARIES
-	@TINYSPLINE_C_LIBRARIES_OUTPUT_NAME@
-	HINTS "@PACKAGE_TINYSPLINE_INSTALL_LIBRARY_DIR@"
-	NO_DEFAULT_PATH)
+include("${CMAKE_CURRENT_LIST_DIR}/tinyspline-targets.cmake")
+set(TINYSPLINE_LIBRARIES tinyspline::tinyspline)
 
 mark_as_advanced(
```

Comment:
Is `mark_as_advanced` still necessary?

Reply:
Looks like @Spacelm [answers](https://github.com/conan-io/conan-center-index/pull/3149) this:

`add ALIAS targets tinyspline::tinyspline and tinysplinecxx::tinysplinecxx (consumers can use those targets, no matter if they rely on add_subdirectory or find_package)`